src/backend/renderer/gles/mod.rs

```diff
@@ -366,6 +368,106 @@ pub enum Capability {
     ColorTransformations,
 }
 
+impl Capability {
+    /// Find all capabilities supported by the driver.
+    pub fn supported_capabilities() -> Vec<Capability> {
```

Comment:
You need to specify a context must be current and possibly return Err if no context is current. The other option is to borrow the context.